sphinx/sphinx.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
 description = 'Generates docs for the python libraries provided in Deephaven Core'
 
 evaluationDependsOn ':docker-runtime-base'
+evaluationDependsOn ':docker-server'
 evaluationDependsOn ':pyclient'
```

Comment:
I'm wondering if we can get rid of the `evaluationDependsOn ':docker-runtime-base'` if we move the doxygen to deephaven/server as well?